src/Microsoft.Framework.Runtime.Roslyn/RoslynProjectReference.cs

```diff
@@ -21,30 +21,20 @@ public class RoslynProjectReference : IRoslynMetadataReference, IMetadataProject
         public RoslynProjectReference(CompilationContext compilationContext)
```

Comment:
Try to avoid random formatting changes in pull requests. It distracts from the actual change.